templates/quickstart-confluence-master.template.yaml

```diff
@@ -1066,8 +1062,8 @@ Resources:
               Service: [ec2.amazonaws.com]
             Action: ['sts:AssumeRole']
       ManagedPolicyArns:
-        - arn:aws:iam::aws:policy/service-role/AmazonEC2RoleforSSM
-        - arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/service-role/AmazonEC2RoleforSSM"
```

Comment:
Is this a fix for govcloud?

Reply:
Yes, the ARN is composed from the AWS partition, so in most cases it is `aws` but in govcloud it is `aws-us-gov` (and also `aws-cn` in China).